components/dashboard/sections/Vendors.tsx

```diff
@@ -435,7 +435,7 @@ const Vendors = ({ accountSlug }: DashboardSectionProps) => {
       <Drawer
         open={isDrawerOpen}
         onClose={closeDrawer}
-        className={cn((vendorDetail || orgDetail) && 'max-w-2xl')}
+        className={cn(vendorDetail || orgDetail ? 'max-w-2xl' : 'max-w-xl')}
```

Comment:
:ice_cream: Sounds like we don't need `cn` in this case

```suggestion
        className={vendorDetail || orgDetail ? 'max-w-2xl' : 'max-w-xl'}
```